azure-pipelines.yml

```diff
@@ -67,7 +67,61 @@ stages:
             twine upload wheelhouse/*
           env:
             TWINE_PASSWORD: $(TWINE_PASSWORD)
-
+    - job: 'Windows_win32_Wheel_Builds'
+      pool: {vmImage: 'vs2017-win2016'}
+#      condition: startsWith(variables['Build.SourceBranch'], 'refs/tags')
+      variables:
+        TWINE_USERNAME: qiskit
+      steps:
+        - checkout: self
+          submodules: true
+        - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+          displayName: Add conda to PATH
+        - bash: |
+            set -x
+            set -e
+            conda update --yes -n base conda
+            conda config --add channels conda-forge
+          displayName: Update Conda
+        - bash: |
+            set -x
+            set -e
+            for version in 3.6 3.7 3.8 ; do
+                conda create --yes --quiet --name qiskit-aer-$version python=$version
+            done
+          displayName: Create Anaconda environments
+          env:
+            CONDA_FORCE_32BIT: 1
+        - bash: |
+            set -x
+            set -e
+            mkdir wheelhouse
+            for version in 3.6 3.7 3.8 ; do
+                source activate qiskit-aer-$version
+                conda install --yes --quiet --name qiskit-aer-$version python=$version numpy cmake virtualenv pip setuptools pybind11 cython scipy
+                python -m pip install -U setuptools wheel
+                python setup.py bdist_wheel -- -G "Visual Studio 15 2017 Win64" -A "Win32"
```

Comment:
I think the syntax is different for VS 2017 and it is just:
`python setup.py bdist_wheel -- -G "Visual Studio 15 2017"`
If it doesn't work with the `-A win32` try with the above syntax